crates/zed/src/languages/regex/brackets.scm

```diff
@@ -0,0 +1,3 @@
+("(" @open ")" @close)
+("[" @open "]" @close)
+("{" @open "}" @close)
```

Comment:
I'm unsure whether to add `<` and `>` because of lookbehind assertions. What do you think about this?